packages/ui/src/components/surfaces/accordion/index.tsx

```diff
@@ -23,7 +32,7 @@ export interface AccordionProps {
     onChange?: (event: React.MouseEvent, expanded: boolean) => void;
     expanded?: boolean;
     className?: { root?: string };
-    children: ReactElement[];
+    children: ReactNode[];
```

Comment:
Does it make sense to have an array here? I think `ReactNode` supports passing stuff like:

```
<div />
<div />
<div />
```

No?

Reply:
I meant to change `ReactElement` to `ReactNode` only `Summary` and `Details`, my bad. I'll revert this change.